sdk/openai/Azure.AI.OpenAI/src/Generated/Docs/OpenAIClient.xml

```diff
@@ -104,92 +104,8 @@ var result = client.GetCompletions("<deploymentId>", completionsOptions);
     <member name="GetCompletions(string,RequestContent,RequestContext)">
     </member>
     <member name="GetChatCompletionsAsync(string,ChatCompletionsOptions,CancellationToken)">
-<example>
```

Comment:
This is removed because I added a check in the generator that only public method will generate samples, and this method is internal therefore this is removed here.